src/hfx_load_balance_methods.F

```diff
@@ -43,7 +43,7 @@ MODULE hfx_load_balance_methods
    PRIVATE
 
    PUBLIC :: hfx_load_balance, &
-             hfx_update_load_balance, &
+             hfx_update_load_balance, reshuffle, &
```

Comment:
maybe rename the procedure to something a little less generic?

Reply:
you can also wait with that, going to replace the whole thing with #1171

Reply:
That would be a nice general tool to have, indeed.